packages/components/src/MultiSelect/DropDownMenu.css

```diff
@@ -0,0 +1,34 @@
+.dropDownMenuContainer {
+  display: flex;
+  position: absolute;
+  z-index: var(--elevation-menu);
+  width: 100%;
+  max-height: 300px;
```

Comment:
Can we replace this with a unit based our space variables? Will allow it to resize on browser zoom in/out.